Dockerfile

```diff
@@ -11,12 +15,14 @@ RUN tar -xf /tmp/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz -C /tmp \
 
 RUN apk add --update git \
     && apk upgrade \
-    && apk add --no-cache ca-certificates
+    && apk add --no-cache ca-certificates \
```

Comment:
One thing I found to be really helpful is to also use the `--progress` option, so it's obvious that things are happening when the container is being compiled.

Reply:
OK